File: string_reverse.js
// string_reverse.js
stringReverse = (str) => {
  let arr = str.split(" ");
  arr.map( word => {
    if(word.length > 4){
      let splitWord = word.split("")
      let newArr = [];
      for(i = splitWord.length - 1; i >= 0; i--){
        newArr.push(splitWord[i])
      }
      console.log(newArr.join(""))
    }else{
      console.log(word)
    }
  })
}

stringReverse("hey fellow warriors")
stringReverse("hello world")

File: string_reverse.rb
# string_reverse.rb
def string_reverse(str)

  arr = str.split(" ") 
  for word in arr
    if word.length > 4
      wordarr = word.split("")
      length = wordarr.length
      index =  length -  1
      length.times do
        print wordarr[index]
        index -= 1
      end
      print " "
    else
      print word + " "
    end
  end
  puts
end


string_reverse("hello world!")
string_reverse("hey fellow warriors")